tests/MyGame/Example/AnyUniqueAliases.go

```diff
@@ -2,18 +2,34 @@
 
 package Example
 
-type AnyUniqueAliases = byte
+import "strconv"
+
+type AnyUniqueAliases byte
+
 const (
 	AnyUniqueAliasesNONE AnyUniqueAliases = 0
-	AnyUniqueAliasesM AnyUniqueAliases = 1
-	AnyUniqueAliasesT AnyUniqueAliases = 2
-	AnyUniqueAliasesM2 AnyUniqueAliases = 3
+	AnyUniqueAliasesM    AnyUniqueAliases = 1
+	AnyUniqueAliasesT    AnyUniqueAliases = 2
+	AnyUniqueAliasesM2   AnyUniqueAliases = 3
 )
 
 var EnumNamesAnyUniqueAliases = map[AnyUniqueAliases]string{
-	AnyUniqueAliasesNONE:"NONE",
-	AnyUniqueAliasesM:"M",
-	AnyUniqueAliasesT:"T",
-	AnyUniqueAliasesM2:"M2",
+	AnyUniqueAliasesNONE: "NONE",
+	AnyUniqueAliasesM:    "M",
+	AnyUniqueAliasesT:    "T",
+	AnyUniqueAliasesM2:   "M2",
 }
 
+var EnumValuesAnyUniqueAliases = map[string]AnyUniqueAliases{
+	"NONE": AnyUniqueAliasesNONE,
+	"M":    AnyUniqueAliasesM,
+	"T":    AnyUniqueAliasesT,
+	"M2":   AnyUniqueAliasesM2,
+}
+
+func (v AnyUniqueAliases) String() string {
+	if s, ok := EnumNamesAnyUniqueAliases[v]; ok {
+		return s
+	}
+	return "AnyUniqueAliases(" + strconv.FormatInt(int64(v), 10) + ")"
```

Comment:
@dnr Did you think about having this generated function return the value from the `EnumNamesAnyUniqueAliases` map? That way we don't need to import `strconv`. If the key is not in the map, we don't want to cause a panic, so we could return a hardcoded string like `AnyUniqueAliases_UNKNOWN`.

Reply:
It does return string values where they exist in the map. The fallback path for unknown values is based on the equivalent in [stringer](https://godoc.org/golang.org/x/tools/cmd/stringer). It provides more information than a fixed string, but if you'd prefer a fixed string that's fine with me too (after all, it's a situation that "shouldn't" happen).

Reply:
Any thoughts on the other comments/questions in the pr?

Reply:
I should've read more closely! You can keep the current logic, since, as you said, it shouldn't happen unless the data is bad.